### ISSUE TITLE
Too many levels of symbolic links, on Linux

### ISSUE THREAD

**Issue by janssen** [michaelrsweet/epm#60]
Not sure *why* the packaging failed, but there is that one error message.  Which I don't understand.  Works OK on MacOS Sierra.  Building on a vagrant box, thus the "/vagrant".

```
Building package as root... into /vagrant
epm: Unable to create "/vagrant/goodstuff-1.0-linux-4.2-amd64/opt/goodstuff/ssl/man/man3/hmac.3" -
     Too many levels of symbolic links
ESP Package Manager v4.4
Copyright 1999-2017 by Michael R Sweet.

EPM is free software and comes with ABSOLUTELY NO WARRANTY; for details
see the GNU General Public License in the file COPYING or at
"http://www.fsf.org/gpl.html".  Report all problems to
"https://github.com/michaelrsweet/epm/issues".

Searching for product information...
Stripping executables in distribution...
Creating Debian goodstuff-1.0-linux-4.2-amd64 distribution...
Creating control file...
Creating postinst script...
Creating conffiles...
Copying temporary distribution files...
Packaging failed!
```

**Comment by michaelrsweet** [comment]
Can you attach the list file for this?

**Comment by janssen** [comment]
22,000 lines, and doubt it would help.  I'm seeing this on a vagrant box Ubuntu 14.04 guest running on an OS X 10.9 host under Parallels.  The offending link is hmac.3 -> HMAC.3.  So I tried it on a regular Ubuntu box (16.04 this time) and see no issue.  I think this is the case-insensitive HFS+ file system of the Mac biting.  I've simply amended my build script to remove the ssl/man directory before running EPM.  I think there's still an issue here, but it's really an OpenSSL issue about case-insensitive file systems.
